### DISCUSSION TITLE
Gamma index comparison code (low dose threshold)

### DISCUSSION THREAD

**Discussion by Pornpapas** [e0404/matRad#770] [Q&A]
Dear developer

I have a question about the gamma index code (matRad_gammaIndex.m) in the tools folder regarding whether the low dose threshold is set? (such as 10% low dose threshold).

Thank you

**Comment by wahln** [comment]
Hi @Pornpapas,
the matRad_gammaIndex function currently has no parameter to set the lower dose threshold, this is definitely something that needs to be added. Currently, the chosen dose criterion is applied as threshold, which is probably not the desired behavior. Here's the line of code in the current dev branch:
https://github.com/e0404/matRad/blob/ba49be40e2c11565a5605799edf916ee909aae90/matRad/planAnalysis/matRad_gammaIndex.m#L174

As a workaround, this line could just be changed to 
`doseIx = cube1 > 0.1*max(cube1(:)) | cube2 > 0.1*max(cube2(:));` 
to achieve the 10% thresholding.

I will turn your discussion into an issue such that it is on our radar for implementing the change and an additional parameter.

**Reply by wahln** [reply]
Issue is here: #771

**Comment by ni484call5** [comment]
Really thanks your effort!

I wonder the meaning of **1.5** in `matRad_gammaIndex` of dev branch
```
neighborhood = 1.5* dist2AgreeMm; % [mm]
searchX = ceil(neighborhood./resolution(1));
searchY = ceil(neighborhood./resolution(2));
searchZ = ceil(neighborhood./resolution(3));
```

Why isn't it just `neighborhood = dist2AgreeMm; % [mm]`?
For example, in a 3%/3mm and 1mm resolution case, with the snippet shown above.
It seems like 3%/5mm, _ceil(1.5* 3mm / 1mm) = 5_.
The comparison region for distance to agreement become larger than requirement.


regards,
Nicholas

**Reply by wahln** [reply]
Choosing a region larger than the requirement does not change the pass-rate. However, it may change the gamma value itself (in the sense that it might find a very close value at larger distance wich might decrease the gamma value. 
Also there is the issue of voxels on the diagonals, but this is probably already captured by "ceil".
Thus, if you are only interested in the pass-rate, you could probably set this to 1. The size of the neighborhood is also a candidate for a changable parameter in a future overhaul (see the created issue #771 due to this discussion).
